module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderContent.kt

```diff
@@ -19,12 +22,16 @@ internal fun ColumnScope.RenderContent(content: List<Content>, state: State) {
 
 @Composable
 internal fun ColumnScope.RenderContent(content: Content, state: State) {
+    val isGone by remember(content, state) { content.isGoneFlow(state) }.collectAsState(false)
```

Comment:
let's use the current `isGone` state as the initial value

```suggestion
    val isGone by remember(content, state) { content.isGoneFlow(state) }.collectAsState(content.isGone(state))
```

Reply:
Ah good call